docs/setup.md

```diff
@@ -9,7 +9,22 @@ This document outlines the setup process for the `Superposition Platform`.
 - Install wasm-pack
     `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
 - Install docker
-- make sure you have dependencies like libpq, openssl, libiconv,etc installed
+- make sure you have dependencies like libpq, openssl, libiconv, diesel_clietc installed
+- To install these dependencies run the following commands -
+    ```bash
+    brew install libpq && brew install openssl && brew install libiconv # Add libpq to the path
+- Instalstdlib-jdk7l diesel_cli
```

Comment:
@Subhash703 - the formatting of the commands seem a little off in the doc.

https://github.com/juspay/superposition/blob/fix/setup-instruction/docs/setup.md

Can we keep all the shell commands listed out separately without indenting them with anything.  The jagged grey highlight boxes on the finally rendered doc looks a little odd.

Reply:
@knutties Could you please check again. Have modified it.